tuning/sft_trainer.py

```diff
@@ -129,9 +129,18 @@ def train(
     )
     
     # load the data by parsing JSON
-    json_dataset = datasets.load_dataset('json', data_files=data_args.data_path)
-    formatted_dataset = json_dataset['train'].map(lambda example : {f"{data_args.dataset_text_field}" : example[f"{data_args.dataset_text_field}"] + tokenizer.eos_token})
-    logger.info(f"Dataset length is {len(formatted_dataset)}")
+    data_files = {"train": data_args.data_path}
+    if data_args.validation_data_path:
+        data_files["validation"] = data_args.validation_data_path
+
+    json_dataset = datasets.load_dataset('json', data_files=data_files)
+    formatted_train_dataset = json_dataset['train'].map(lambda example : {f"{data_args.dataset_text_field}" : example[f"{data_args.dataset_text_field}"] + tokenizer.eos_token})
+    logger.info(f"Dataset length is {len(formatted_train_dataset)}")
+
+    formatted_validation_dataset = None
```

Comment:
Note that passing in None is okay for [eval_dataset](https://huggingface.co/docs/trl/sft_trainer#trl.SFTTrainer.eval_dataset), tested running this and eval_dataset is ignored. None is the default value